src/chainlink/oracle/ChainlinkOracleL2Impl.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.17;
+
+import {ChainlinkOracleImpl} from "./ChainlinkOracleImpl.sol";
+import {AggregatorV3Interface} from "chainlink/interfaces/AggregatorV3Interface.sol";
+import {QuoteParams} from "splits-utils/LibQuotes.sol";
+
+/// @title Chainlink L2 Oracle Implementation
+/// @author 0xSplits
+/// @notice A clone-implementation of an oracle using Chainlink
+/// @dev This contract uses token = address(0) to refer to ETH.
+contract ChainlinkOracleL2Impl is ChainlinkOracleImpl {
+    /// -----------------------------------------------------------------------
+    /// errors
+    /// -----------------------------------------------------------------------
+
+    error SequencerDown();
+    error GracePeriodNotOver();
+
+    /// -----------------------------------------------------------------------
+    /// storage
+    /// -----------------------------------------------------------------------
+
+    /// -----------------------------------------------------------------------
+    /// storage - constants & immutables
+    /// -----------------------------------------------------------------------
+
+    AggregatorV3Interface public immutable sequencerFeed;
+    uint256 constant GRACE_PERIOD_TIME = 1 hours;
```

Comment:
any strong arguments for/against making this storage/mutable?

Reply:
This is the recommended grace period by the chainlink team, I would be okay with making it mutable I just don't know what would go into deciding the grace period at the moment

Reply:
eh.. probably fine to leave as-is. Doesn't seem like much of a gain from making it immutable or regular storage